src/Request/ParamConverter/DoctrineQueryStringConverter.php

```diff
@@ -0,0 +1,308 @@
+<?php
+
+namespace AppBundle\Request\ParamConverter;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+use Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\ParamConverterInterface;
+use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
+use Symfony\Component\ExpressionLanguage\SyntaxError;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\HttpFoundation\Request;
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Doctrine\ORM\NoResultException;
+
+/**
+ * Puts specific attribute parameter to query parameters and does the same job as DoctrineParamConverter.
+ */
+class DoctrineQueryStringConverter implements ParamConverterInterface
```

Comment:
what do you think about this solution:
```php
class QueryStringParamConverter extends DoctrineParamConverter
{
    public function apply(Request $request, ParamConverter $configuration)
    {
        $name = $configuration->getName();

        if (!$request->query->has($name)) {
            throw new \InvalidArgumentException($name.' param is missing in the request');
        }

        $request->attributes->set($name, $request->query->get($name));

        return parent::apply($request, $configuration);
    }
}
```

with a service tag:
```
<tag name="request.param_converter" converter="querystring" priority="false" />
```

`priority=false` allow use this converter via `converter="querystring"` annotation and avoid adding is in the stack of all converters.

Reply:
Changed.